pom.xml

```diff
@@ -68,7 +68,16 @@
         <dependency>
             <groupId>edu.illinois.cs</groupId>
             <artifactId>testrunner-maven-plugin</artifactId>
-            <version>1.0</version>
+            <version>1.1</version>
+        </dependency>
+
+        <!-- Need below dependency to compile after upgrading
```

Comment:
Is this dependency still needed?

Reply:
Yes, otherwise it fails to compile. I think it's a legacy issue and I didn't look deep into it. Maybe someone needs to release a version for that dependency.

Reply:
It seems to be related to the class `edu.illinois.cs.dt.tools.runner.data.TestResult`, where it depends on XStream. That was related to some code we were experimenting with in testrunner but have since removed from there. This `TestResult` is actually not used anywhere and should just be deleted.

Reply:
OK, I will delete it if so.

Reply:
Done